### PR TITLE
vonk: releasenotes for 3.3.0

### PR DESCRIPTION
**Pull request by cknaap**

### CODE REVIEW

File: vonk/releasenotes/releasenotes.rst [hunk 1]
```diff
@@ -17,6 +17,56 @@ Upgrading Vonk
...
+#. The generated CapabilityStatement for R4 failed constraint cpb-14.
+#. Content negotiation favoured a mediatype with quality < 1 over a mediatype without quality. But the default value is 1, so the latter is now favoured. 
+#. :ref:`feature_validation_instance` did not account for the informationmodel (aka FHIR version) of the resource.
+
+Plugins & Facade
+^^^^^^^^^^^^^^^^
+
+#. :ref:`Document Operation <vonk_plugins_documentoperation>` 
+   
+   * Has been upgraded to Vonk 3.2.0.
```
Comment (alexzautke) [inline vonk/releasenotes/releasenotes.rst]: We also fixed the Bundle.id returned from $document
https://github.com/FirelyTeam/Vonk.Plugin.DocumentOperation/commit/c08b7d3d169aa4eb36836b47e7358b06b5f4a71f

File: vonk/faq.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Searchparameter errors for composite parameters
+-----------------------------------------------
+
+When importing specification.zip for various FHIR versions, Vonk will report errors like these:
+
+::
+
+   Composite SearchParameter 'CodeSystem.context-type-quantity' doesn't have components.
+
+Vonk checks whether all the components of a composite searchparameter are present. So this error can be correct for a searchparameters that is not well defined. But the implementation of this check seems to have an error so too many composite parameters are reported as faulty.
```
Comment (vadi2) [inline vonk/faq.rst]: Bad grammar 🔨

File: vonk/features/auditing.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The OutputTemplate listed here contains all the properties that can be logged:
+
+* RequestId: unique id of this request, use this to correlate request and response
+* Ip: IP Address of the client
+* ConnectionId: use this to correlate requests from the same client
+* UserId: user id from the JWT token (if present)
+* UserName: user name from the JWT token (if present)
+* Path: request url
+* Action: interaction that was request (like instance_read or type_search)
```
Comment (vadi2) [inline vonk/features/auditing.rst]: Requested

File: vonk/features/auditing.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* Resource: resourcetype involved
+* ResourceKey: key of the resource involved (type/id)
```
Comment (vadi2) [inline vonk/features/auditing.rst]: This needs more explanation - key is not a FHIR term

**Pull request merged**